projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -277,7 +309,12 @@ static function load_wrapper_styles() {
 		';
 		wp_add_inline_style( 'dops-css', $custom_css );
 	}
-
+	/**
+	 * Build footer, content, and footer for admin page.
+	 *
+	 * @param string $callback The callback sent to the Jetpack_Admin_Page::wrap_ui method. Callback is responsible for any needed escaping.
+	 * @param array  $args The arguments sent to the Jetpack_Admin_Page::wrap_ui method.
```

Comment:
These descriptions aren't very descriptive. Maybe something like this instead?
```suggestion
	 * @param string $callback Callback to produce the content of the page. The callback is responsible for any needed escaping.
	 * @param array  $args Options for the wrapping. Also passed to the `jetpack_admin_pages_wrap_ui_after_callback` action.
	 *   - is-wide: (bool) Set the "is-wide" class on the wrapper div, which increases the max width. Default false.
	 *   - show-nav: (bool) Whether to show the navigation bar at the top of the page. Default true.
```

Reply:
Sounds much better :-)